index.js

```diff
@@ -10,7 +10,8 @@ const defaultConfig = {
     readme: 'https://github.com/duckduckgo/privacy-configuration',
     version: Date.now(),
     features: {},
-    unprotectedTemporary: []
+    unprotectedTemporary: [],
+    experimentalVariants: {}
```

Comment:
Wonder if we should lock this to only Android right now? I'd just be afraid of blowing up other platforms.

Reply:
Fine by me, removed